src/Lorentz/Contracts/Spec/FA2Interface.hs

```diff
@@ -0,0 +1,308 @@
+-- SPDX-FileCopyrightText: 2020 Tocqueville Group
+--
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+-- | FA2 interface specification.
+-- https://gitlab.com/tzip/tzip/-/blob/667f925471728bb8e81fbd30b93a171e401b6dc1/proposals/tzip-12/tzip-12.md
+--
+module Lorentz.Contracts.Spec.FA2Interface
```

Comment:
Btw, why are you not using this module from `morley-ledgers`? I think we can use it here as is, because it matches FA2 directly. And it's added as a dependency in #13 anyway.